src/model/document/document.ts

```diff
@@ -266,6 +286,39 @@ variableManager: shallowReadonly(
     return element as any
   }
 
+  function getElementsByType<T extends keyof TElements>(typeName: T): QReturnType<TElements[T]['elementType']>[] | undefined {
+    let elements = elementList.elements.filter((e) => e.typeName == typeName)
+
+    // Yeah, Typescript really does dislike this XD
+    return elements as any[]
+  }
+
+  function serializeDocument() {
+    var serializedData: SerializedDataType = {
```

Comment:
Change that to a `let`, I guess. `var` is old-fashioned. 

(The difference is that `var` will hoist the variable name and thus magically place it at the top of the function. `let` won't to that)

Reply:
I like that better